examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
+       "      <td>0.001753</td>\n",
+       "      <td>0.000465</td>\n",
+       "      <td>0.000345</td>\n",
+       "      <td>0.000093</td>\n",
+       "      <td>0.01</td>\n",
+       "      <td>{'C': 0.01}</td>\n",
+       "      <td>0.657143</td>\n",
+       "      <td>0.636364</td>\n",
+       "      <td>0.65625</td>\n",
+       "      <td>0.649919</td>\n",
+       "      <td>0.009592</td>\n",
+       "      <td>5</td>\n",
+       "      <td>0.646154</td>\n",
+       "      <td>0.656716</td>\n",
+       "      <td>0.647059</td>\n",
+       "      <td>0.649976</td>\n",
+       "      <td>0.004780</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>1</td>\n",
+       "      <td>0.001419</td>\n",
+       "      <td>0.000217</td>\n",
+       "      <td>0.000303</td>\n",
+       "      <td>0.000024</td>\n",
+       "      <td>0.1</td>\n",
+       "      <td>{'C': 0.1}</td>\n",
+       "      <td>0.800000</td>\n",
+       "      <td>0.757576</td>\n",
+       "      <td>0.75000</td>\n",
+       "      <td>0.769192</td>\n",
+       "      <td>0.022003</td>\n",
+       "      <td>4</td>\n",
+       "      <td>0.753846</td>\n",
+       "      <td>0.820896</td>\n",
+       "      <td>0.808824</td>\n",
+       "      <td>0.794522</td>\n",
+       "      <td>0.029181</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>2</td>\n",
+       "      <td>0.001272</td>\n",
+       "      <td>0.000069</td>\n",
+       "      <td>0.000260</td>\n",
+       "      <td>0.000012</td>\n",
+       "      <td>1</td>\n",
+       "      <td>{'C': 1.0}</td>\n",
+       "      <td>1.000000</td>\n",
+       "      <td>0.818182</td>\n",
+       "      <td>0.93750</td>\n",
+       "      <td>0.918561</td>\n",
+       "      <td>0.075425</td>\n",
+       "      <td>3</td>\n",
+       "      <td>0.938462</td>\n",
+       "      <td>0.985075</td>\n",
+       "      <td>0.955882</td>\n",
+       "      <td>0.959806</td>\n",
+       "      <td>0.019231</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>3</td>\n",
+       "      <td>0.001333</td>\n",
+       "      <td>0.000064</td>\n",
+       "      <td>0.000272</td>\n",
+       "      <td>0.000030</td>\n",
+       "      <td>10</td>\n",
+       "      <td>{'C': 10.0}</td>\n",
+       "      <td>1.000000</td>\n",
+       "      <td>0.818182</td>\n",
+       "      <td>1.00000</td>\n",
+       "      <td>0.939394</td>\n",
+       "      <td>0.085710</td>\n",
+       "      <td>2</td>\n",
+       "      <td>0.969231</td>\n",
+       "      <td>0.985075</td>\n",
+       "      <td>0.970588</td>\n",
+       "      <td>0.974965</td>\n",
+       "      <td>0.007170</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>4</td>\n",
+       "      <td>0.001271</td>\n",
+       "      <td>0.000034</td>\n",
+       "      <td>0.000240</td>\n",
+       "      <td>0.000004</td>\n",
+       "      <td>100</td>\n",
+       "      <td>{'C': 10
```

Comment:
For some reason, I see the following warnings when running this cell:
```
/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split0_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
  warnings.warn(*warn_args, **warn_kwargs)
/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split1_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
  warnings.warn(*warn_args, **warn_kwargs)
/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('split2_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
  warnings.warn(*warn_args, **warn_kwargs)
/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('mean_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
  warnings.warn(*warn_args, **warn_kwargs)
/srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/utils/deprecation.py:125: FutureWarning: You are accessing a training score ('std_train_score'), which will not be available by default any more in 0.21. If you need training scores, please set return_train_score=True
  warnings.warn(*warn_args, **warn_kwargs)
```

If there's anything we can do about this, that would be great.

Reply:
It's hard to comment on a raw ipynb file. The line I meant was `pd.DataFrame(grid_search_results)`, to be clear.

Reply:
Yeah, this would go away once we merge in the scikit-learn 0.21.3 branch since we would no longer generate these fields.